src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+=============================
+SageMaker Workflow in Airflow
```

Comment:
Do you need to change the master readme and add a workflow section?

Reply:
Good idea! One section added to master readme and linked to this one.